packages/storybook/stories/components/Textarea.stories.tsx

```diff
@@ -0,0 +1,209 @@
+    },
+    spellCheck: {
+      description: 'Spellcheck',
+      control: { type: 'select' },
+      options: ['', 'false', 'true'],
+    },
+  },
+  args: {
+    dir: 'auto',
+    disabled: false,
+    focus: false,
+    focusVisible: false,
+    hover: false,
+    invalid: false,
+    readOnly: false,
+    required: false,
+    children: 'The Quick Brown Fox Jumps Over The Lazy Dog',
+  },
+  tags: ['autodocs'],
+  parameters: {
+    status: {
+      type: 'WORK IN PROGRESS',
+    },
+    tokensPrefix: 'utrecht-textarea',
+    tokens,
+    tokensDefinition,
+    docs: {
+      description: {
+        component: readme,
+      },
+    },
+  },
+} satisfies Meta<typeof Textarea>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    children: 'The Quick Brown Fox Jumps Over The Lazy Dog',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: `Styling via the \`.utrecht-textarea\` class name.`,
+      },
+    },
+  },
+};
+
+export const DefaultValue: Story = {
+  args: {
+    ...Default.args,
+  },
+  render: (args) => {
+    return <Textarea defaultValue={args.children} />;
+  },
+};
+
+export const PlaceholderRightToLeft: Story = {
+  name: 'Placeholder (right-to-left)',
+  args: {
+    ...Default.args,
+    placeholder: 'اكتب رسالتك',
+    children: '',
+  },
+  decorators: [arabicDecorator],
+};
+
+export const RightToLeft: Story = {
+  args: {
+    ...Default.args,
+    children:
+      'لكن لا بد أن أوضح لك أن كل هذه الأفكار المغلوطة حول استنكار  النشوة وتمجيد الألم نشأت بالفعل، وسأعرض لك التفاصيل لتكتشف حقيقة وأساس تلك السعادة البشرية، فلا أحد يرفض أو يكره أو يتجنب الشعور بالسعادة، ولكن بفضل هؤلاء الأشخاص الذين لا يدركون بأن السعادة لا بد أن نستشعرها بصورة أكثر عقلانية ومنطقية فيعرضهم هذا لمواجهة الظروف الأليمة، وأكرر بأنه لا يوجد من يرغب في الحب ونيل المنال ويتلذذ بالآلام، الألم هو الألم ولكن نتيجة لظروف ما قد تكمن السعاده فيما نتحمله من كد وأسي.',
+  },
+  decorators: [arabicDecorator],
+};
+
+export const Spellcheck: Story = {
+  args: {
+    ...Default.args,
+    spellCheck: true,
+  },
+};
+
+export const Disabled: Story = {
+  args: {
+    ...Default.args,
+    disabled: true,
+  },
+};
+
+export const Invalid: Story = {
+  args: {
+    ...Default.args,
+    invalid: true,
+  },
+};
+
+export const ReadOnly: Story = {
+  args: {
+    ...Default.args,
+    readOnly: true,
+  },
+};
+
+export const Required: Story = {
+  args: {
+    ...Default.args,
+    required: true,
+  },
+};
+
+export const Focus: Story = {
+  args: {
+    ...Default.args,
+    focus: true,
+    focusVisible: true,
+  },
+  parameters: {
+    ...Default.parameters,
+    docs: {
+      ...Default.parameters?.['docs'],
+      source: {
+        code: `<textarea class="utrecht-textarea utrecht-textarea--html-textarea utrecht-textarea--focus utrecht-textarea--focus-visible">
+                 The Quick Brown Fox Jumps Over The Lazy Dog
+               </textarea>`,
```

Comment:
The code indentation is way off. If you implement the class name as component rendering like we do in `Mark.stories.tsx`, the code preview will be automatically formatted nicely using Prettier.

<img width="1024" alt="Screenshot 2023-08-31 at 08 36 06" src="https://github.com/nl-design-system/utrecht/assets/30694/59feb40a-c792-4854-8ff2-60dec5b6cbf4">